app.rb

```diff
@@ -47,6 +47,54 @@
 end
 
 post '/portal' do
+  if params[:intent] == "audit_logs"
+    WorkOS::AuditLogs.create_event(
+      organization: params[:organization],
+      event: {
+        action: "user.signed_in",
+        occurred_at: Time.now.iso8601(3),
+        actor: {
+          id: "user_01GBNJC3MX9ZZJW1FSTF4C5938",
+          name: Faker::Name.name,
+          type: "user"
+        },
+        targets: [
+          {
+            id: "team_01GBNJD4MKHVKJGEWK42JNMBGS",
+            type: "team"
+          }
+        ],
+        context: {
+          location: request.ip,
+          user_agent: request.user_agent
+        }
+      }
+    )
+
+    WorkOS::AuditLogs.create_event(
+      organization: params[:organization],
+      event: {
+        action: "user.signed_out",
+        occurred_at: Time.now.iso8601(3),
+        actor: {
+          id: "user_01GBNJC3MX9ZZJW1FSTF4C5938",
+          name: Faker::Name.name,
+          type: "user"
+        },
+        targets: [
+          {
+            id: "team_01GBNJD4MKHVKJGEWK42JNMBGS",
+            type: "team"
+          }
+        ],
+        context: {
+          location: request.ip,
+          user_agent: request.user_agent
+        }
+      }
+    )
```

Comment:
Mind if we collapse these two events down a bit?

```suggestion
    ["user.signed_in", "user.signed_out"].each do |action|
      WorkOS::AuditLogs.create_event(
        organization: params[:organization],
        event: {
          action: action,
          occurred_at: Time.now.iso8601(3),
          actor: {
            id: "user_01GBNJC3MX9ZZJW1FSTF4C5938",
            name: Faker::Name.name,
            type: "user"
          },
          targets: [
            {
              id: "team_01GBNJD4MKHVKJGEWK42JNMBGS",
              type: "team"
            }
          ],
          context: {
            location: request.ip,
            user_agent: request.user_agent
          }
        }
      )
    end
```